Tasks/ANT/task.json

```diff
@@ -45,7 +50,25 @@
             "type": "string", 
             "label": "Target(s)", 
             "defaultValue":"", "required":false
-        },        
+        }, 
+        {
+            "name":"publishJUnitResults",
+            "type":"boolean",
+            "label":"Publish to VSO/TFS",
+            "required":true,
+            "defaultValue":"true",
+            "groupName":"junitTestResults",
+            "helpMarkDown":"Select this option to publish JUnit Test results produced by the Ant build to VSO/TFS. Each test result file matching `TEST-*.xml` pattern will be published as a test run in VSO/TFS."
+        }, 
+        {
+            "name": "testResultsFiles",
+            "type": "filePath",
+            "label": "Test Results Files",
+            "defaultValue": "**\\TEST-*.xml",
+            "required": true,
+            "groupName":"junitTestResults",
+            "helpMarkDown": "Test results files path.  Wildcards can be used.  For example, `**\\*TEST-*.xml` for all xml files whose name starts with TEST-."
+        },            
         {
             "name":"jdkVersion",
             "type":"pickList",
```

Comment:
You need to build to get the task.loc.json file generated. It should be checked in with the changes.